### DISCUSSION TITLE
Contribute to Open-source

### DISCUSSION THREAD

**Discussion by Tanvirmehedi** [immersive-translate/immersive-translate#771] [Help]
### 概述 | Summary

I did not see any contribution guidelines in the immersive-translation repository. I saw that many people are keeping frocks but there is no pull request. Does that mean this extension is not for open-source contributions?

**Comment by kitety** [comment]
Yes， it is not open source.
Some link has pointed out this.
Link: https://www.v2ex.com/t/961178
